cloud/deployment/deployment.go

```diff
@@ -69,6 +69,7 @@ var (
 	tickNum                    = 10
 	timeoutNum                 = 180
 	dedicatedDeploymentRequest = astroplatformcore.UpdateDedicatedDeploymentRequest{}
+	dagDeployEnabled           bool
```

Comment:
what was the reasoning behind putting it up here, it seems to fit the style of code the way we had it before

Reply:
We need it exist outside of the function so we can assert that it is the right value in the test